python_modules/libraries/dagster-mysql/dagster_mysql/schedule_storage/schedule_storage.py

```diff
@@ -83,59 +90,61 @@ def optimize_for_dagit(self, statement_timeout, pool_recycle):
         )
 
     @property
-    def inst_data(self):
+    def inst_data(self) -> Optional[ConfigurableClassData]:
         return self._inst_data
 
     @classmethod
-    def config_type(cls):
+    def config_type(cls) -> UserConfigSchema:
         return mysql_config()
 
     @staticmethod
-    def from_config_value(inst_data, config_value):
+    def from_config_value(
+        inst_data: Optional[ConfigurableClassData], config_value: MySqlStorageConfig
+    ) -> "MySQLScheduleStorage":
         return MySQLScheduleStorage(
             inst_data=inst_data, mysql_url=mysql_url_from_config(config_value)
         )
 
     @staticmethod
-    def wipe_storage(mysql_url):
-        engine = create_engine(mysql_url, isolation_level="AUTOCOMMIT", poolclass=db.pool.NullPool)
+    def wipe_storage(mysql_url: str) -> None:
+        engine = create_engine(mysql_url, isolation_level="AUTOCOMMIT", poolclass=db_pool.NullPool)
         try:
             ScheduleStorageSqlMetadata.drop_all(engine)
         finally:
             engine.dispose()
 
     @staticmethod
-    def create_clean_storage(mysql_url):
+    def create_clean_storage(mysql_url: str) -> "MySQLScheduleStorage":
         MySQLScheduleStorage.wipe_storage(mysql_url)
         return MySQLScheduleStorage(mysql_url)
 
-    def connect(self, run_id=None):  # pylint: disable=arguments-differ, unused-argument
+    def connect(self, run_id: Optional[str] = None) -> ContextManager[Connection]:
         return create_mysql_connection(self._engine, __file__, "schedule")
 
     @property
-    def supports_batch_queries(self):
+    def supports_batch_queries(self) -> bool:
         if not self._mysql_version:
             return False
 
         return parse_mysql_version(self._mysql_version) >= parse_mysql_version(
             MINIMUM_MYSQL_BATCH_VERSION
         )
 
-    def get_server_version(self):
+    def get_server_version(self) -> Optional[str]:
         rows = self.execute("select version()")
         if not rows:
             return None
 
         return rows[0][0]
 
-    def upgrade(self):
+    def upgrade(self) -> None:
         alembic_config = mysql_alembic_config(__file__)
-        run_alembic_upgrade(alembic_config, self._engine)
+        run_alembic_upgrade(alembic_config, self._engine)  # type: ignore  # shouldn't this be a connection
```

Comment:
who is this comment directed to?

Reply:
Anyone who tries to fix the type error here-- reworded slightly.